src/votekit/ballot_generator.py

```diff
@@ -644,16 +683,37 @@ class CambridgeSampler(BallotGenerator):
     def __init__(
         self,
         slate_to_candidates=None,
-        bloc_crossover_rate=None,
+        cohesion_parameters=None,
         path: Optional[Path] = None,
+        majority_name: Optional[str] = "W",
+        minority_name: Optional[str] = "C",
         **data,
     ):
 
         # Call the parent class's __init__ method to handle common parameters
         super().__init__(**data)
 
         self.slate_to_candidates = slate_to_candidates
-        self.bloc_crossover_rate = bloc_crossover_rate
+        self.cohesion_parameters = cohesion_parameters
+
+        # changing names to match historical data
+        majority_bloc = [bloc for bloc, prop in self.bloc_voter_prop.items() if prop>=.5][0]
+        minority_bloc = [bloc for bloc in self.bloc_voter_prop.keys() \
+                                              if bloc != majority_bloc][0]
+        
+        cambridge_names = {majority_bloc: majority_name, minority_bloc: minority_name}
+
+        self.slate_to_candidates = {cambridge_names[b]: self.slate_to_candidates[b] \
```

Comment:
This is more a style thing but you don't need the line break `\` since everything is inside the comprehension\brackets `{}`